src/fetch-wrapper.ts

```diff
@@ -38,20 +38,16 @@ export default function fetchWrapper(
 
   return fetch(
     requestOptions.url,
-    Object.assign(
-      {
-        method: requestOptions.method,
-        body: requestOptions.body,
-        headers: requestOptions.headers as HeadersInit,
-        redirect: requestOptions.redirect,
-        // duplex must be set if request.body is ReadableStream or Async Iterables.
-        // See https://fetch.spec.whatwg.org/#dom-requestinit-duplex.
-        ...(requestOptions.body && { duplex: "half" }),
-      },
-      // `requestOptions.request.agent` type is incompatible
-      // see https://github.com/octokit/types.ts/pull/264
-      requestOptions.request as any,
-    ),
+    Object.assign({
```

Comment:
Is the `Object.assign()` necessary here? Could we not simply use an object

Reply:
You're correct, I've done so now.